README.md

```diff
@@ -248,8 +248,15 @@ QAndQDotBounds waits for a biorbd model and returns a structure with the minimal
 How convenient!
 ```python
 x_bounds = QAndQDotBounds(biorbd_model)
-```
-Then, override the first and last column to be 0, that is the sideways and rotation to be null for both the position and the velocities
+
+
+The first element of x_bounds is the degree of freedom declared in order in the bio.Mod file.
```

Comment:
first elements ... are ... as declared? (plural)
Please note that this is not always the case as a user can change the order as he is pleased... So I am not sure it make sense to write this here.. Maybe specify that it is the default settings for TORQUE_DRIVEN dynamics?

`in order`? `in the same order as in`?

Reply:
Did you mean .bioMod?

Reply:
Also, please note that the exact same idea is said litteraly two sentences before (`To define that, it would be nice to first define boundary constraints on the position (*q*) and velocities (*qdot*) that match those in the bioMod file and to apply them at the very beginning, the very end and all the intermediate nodes as well.`)
Maybe update this sentence instead of producing a new one

Reply:
"In the default settings for TORQUE_DRIVEN dynamics, the first elements of x_bounds are the degrees of freedom  (*q*)  and their velocities (*qdot*) that match those in the bioMod file. The time is discretized in nodes wich are the second elements declared in x_bounds
If you have more than one phase, we shall have x_bound[phase][degrees of freedom, nodes]"